src/app/dashboard/dashLayout.tsx

```diff
@@ -1,5 +1,7 @@
 import Image from 'next/image';
 import React from 'react'
+import Logo from '../../../public/Logo White.png'
```

Comment:
while its a good practice to import this way. Cuz it enables refactoring support.

But in this case, since the image is in public directory. It automatically served. So there is no need u could directly pass the ref : "/Logo White.png".